apps/admin-portal/src/components/users/add-user-groups.tsx

```diff
@@ -222,13 +222,20 @@ export const AddUserGroup: FunctionComponent<AddUserGroupPropsInterface> = (
                 removeItems={ removeGroups }
                 handleUnelectedListSearch={ handleUnselectedListSearch }
                 handleSelectedListSearch={ handleSelectedListSearch }
+                unselectedSegmentTestId="user_mgt_add_user_wizard_modal_unselected_groups_segment"
```

Comment:
Shall we pass just one `testid` to this component and suffix it to inner components?

Reply:
Resolved with c7d96ec